src/tests/ports/owners.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { ISubgraphComponent } from '@well-known-components/thegraph-component'
+import { FetchOptions } from '../../ports/merger/types'
+import { createOwnersComponent } from '../../ports/owner/component'
+import {
+  IOwnerDataComponent,
+  OwnerFragment,
+  OwnersFilters,
+  OwnersSortBy,
+} from '../../ports/owner/types'
+
+let collectionSubgraphMock: ISubgraphComponent
+let getOwnersQueryMock: jest.Mock
+let ownersComponentMock: IOwnerDataComponent
+let queryFilters: FetchOptions<OwnersFilters, OwnersSortBy>
+
+describe('when fetching owners', () => {
+  beforeEach(() => {
+    getOwnersQueryMock = jest.fn()
+
+    collectionSubgraphMock = {
+      query: getOwnersQueryMock,
+    }
+
+    ownersComponentMock = createOwnersComponent({
+      subgraph: collectionSubgraphMock,
+    })
+
+    queryFilters = {
+      contractAddress: 'contractAddress',
+      itemId: 'itemId',
+      first: 10,
+      skip: 10,
+      sortBy: OwnersSortBy.ISSUED_ID,
+    }
+  })
+
+  afterEach(() => {
+    getOwnersQueryMock.mockRestore()
```

Comment:
https://jestjs.io/docs/mock-function-api#mockfnmockrestore If I'm not mistaken, mockRestore only works for spyOn and not for jest.fn().

For `getOwnersQueryMock = jest.fn()` There is no need to restore anything I believe, as it is being recreated eveytime